docs/faq.md

```diff
@@ -12,6 +12,28 @@ hr {
 }
 </style>
 
+<hr id="operation-buffering-timed-out" />
+
+<a class="anchor" href="#operation-buffering-timed-out">**Q**</a>. Operation `...` timed out after 10000 ms. What gives?
+
+**A**. At its core, this issue stems from being unable to connect to mongoose. If you are using an application, like a bot,
```

Comment:
"from Mongoose being unable to connect to MongoDB."

Also, please don't mention bots or anything like that.

Reply:
From the times I've seen this issue report it is usually someone building a discord bot so I thought it would be helpful to give a relevant example.